public/modules/home/views/home.client.view.html

```diff
@@ -33,149 +18,112 @@
-                  </div>
+      </section>
+  </article>
+
+  <article class="pricing">
+      <a href="#account" class="btn btn-circle bottom-center" scroll-to-element>
+          <i class="fa fa-angle-down animated"></i>
+      </a>
+      <section class="container">
+          <div class="row top-buffer">
+              <div class="intro col-sm-12">
+                  <h2 class="text-center">Simple Pricing</h1>
+                  <h4 class="text-center">Start with a simple $5 per month. As you grow, you pay only for the resources you use</h2>
               </div>
           </div>
-          <div class="row">
-              <div class="col-xs-12 col-sm-4 col-md-4 col-lg-4">
-                  <div class="content-panel">
-                      <!-- Content panel in every span for padding -->
-                      <div class="boxgrid" box-grid-effects="slideright">
-                          <img class="cover" ng-src="modules/home/img/filler/11.jpg" alt="" />
-                          <h3>Project</h3>
-                          <p>Aenean tempus dui<br/><a class="soft-hover" ng-href="portfolioitem">More Work</a></p>
-                      </div>
-                  </div>
-              </div>
-              <div class="col-xs-12 col-sm-4 col-md-4 col-lg-4">
-                  <div class="content-panel">
-                      <!-- Content panel in every span for padding -->
-                      <div class="boxgrid" box-grid-effects="slideright">
-                          <img class="cover" ng-src="modules/home/img/filler/10.jpg" alt="" />
-                          <h3>Project</h3>
-                          <p>Aenean tempus dui<br/><a class="soft-hover" ng-href="portfolioitem">More Work</a></p>
-                      </div>
-                  </div>
+
+          <div class="row top-buffer">
+              <div class="intro col-sm-2 fa-icon-container">
+                <i class="fa fa-fire big"></i>
               </div>
-              <div class="col-xs-12 col-sm-4 col-md-4 col-lg-4">
-                  <div class="content-panel">
-                      <!-- Content panel in every span for padding -->
-                      <div class="boxgrid" box-grid-effects="slideright">
-                          <img class="cover" ng-src="modules/home/img/filler/9.jpg" alt="" />
-                          <h3>Project</h3>
-                          <p>Aenean tempus dui<br/><a class="soft-hover" ng-href="portfolioitem">More Work</a></p>
-                      </div>
+              <div class="intro col-sm-10">
+                  <h3>Get Started with Ignition</h3>
+                  <div class="col-sm-12">
+                      <p>To get you started, The Digital Garage has put together a base package for just $5 month.</p>
+                      <p>For $5 month, you get: 512 MB / 1 cpu, 10 GB of SSD Disk and 1TB of Transfer</p>
                   </div>
               </div>
           </div>
-          <div class="row">
```

Comment:
This two Items, Get Started with Ignition and the following are too close, one to another, please use margin top or separate them a bit more.